lesson4/hometask_frameq.rb

```diff
@@ -0,0 +1,29 @@
+class LazyHash < Hash
+  def method_missing(name)
+    key = name.to_s
+    super.method_missing(key) unless self.key?(key)
+    if self[key].class == Proc
```

Comment:
**Do not check classes in Ruby directly.**
Use `#respond_to?` whenever it's possible or `#is_a?` / `#kind_of?` if you **really** need to check class type, not allowed interface.